src/main/scala/ru/chermenin/spark/sql/execution/streaming/state/RocksDbStateStoreProvider.scala

```diff
@@ -701,12 +726,50 @@ object RocksDbStateStoreProvider {
     cacheBuilderWithOptions.build[UnsafeRow, String](loader)
   }
 
-  private def setTTL(conf: Map[String, String]): Int =
+  /**
+    * Creates a mapping of Streaming Query and its expiry timeout (seconds).
+    * For backward compatibility, an additional entry is done for [[UNNAMED_QUERY]]'s
+    *
+    * The timeout value for [[UNNAMED_QUERY]]'s is set by the value of [[STATE_EXPIRY_SECS]]
+    *
+    * @param stateStoreConf state store config map set on [[SparkConf]]
+    * @return mapping of queryName -> expirySecs
+    */
+
+  private def getExpirationByQuery(stateStoreConf: Map[String, String]): Map[String, Int] =
+    stateStoreConf
+      .filter(_._1.startsWith(s"$STATE_EXPIRY_SECS."))
```

Comment:
Please, avoid using `_1` and other similar methods to keep readability. It can be implemented the following way:
```
.filterKeys(_.startsWith(s"$STATE_EXPIRY_SECS."))
.map { case (key, value) => key.replace(s"$STATE_EXPIRY_SECS.", "") -> getTTL(value) }
.+(UNNAMED_QUERY -> stateStoreConf.getOrElse(STATE_EXPIRY_SECS, DEFAULT_STATE_EXPIRY_SECS).toInt)
```
And I think the `->` sugar is better than using tuples for maps.

Reply:
thats actually correct